library/Util.re

```diff
@@ -1,64 +1,25 @@
 open Ppxlib;
 
-let validateCssStr = (~loc, cssStr) => {
+let errorInCssString = cssStr => {
   let classes = cssStr |> Tablecloth.String.split(~on=" ");
 
   TailwindData.classNames
-  |> List.iteri((i, n) =>
-       switch (classes |> List.find_opt(c => c == n)) {
-       | Some(_) => ()
-       | None => Location.raise_errorf("\"%s\" is not a valid class name", n)
-       }
+  |> List.find_opt(n => !(classes |> List.exists(c => c == n)))
+  |> Tablecloth.Option.map(~f=n =>
+       Printf.sprintf("\"%s\" is not a valid class name", n)
```

Comment:
You should calculate and return a more precise location here.

Reply:
To report multiple exceptions you can use https://github.com/ocaml-ppx/ppxlib/blob/8d2ce31611569b1342df9ecd195264f556892344/src/location.ml#L27

Reply:
Actually a better way is to insert `[%%ocaml.error "Error message here"]` in the resulting expression instead of throwing:
https://github.com/ocaml-ppx/ppx/issues/20